lake/seekindex/writer.go

```diff
@@ -22,18 +22,20 @@ func NewWriter(w zio.Writer) *Writer {
 	}
 }
 
-func (w *Writer) Write(key zed.Value, offset int64) error {
+func (w *Writer) Write(key zed.Value, number uint64, offset int64) error {
```

Comment:
Be consistent with the Zed record column name?
```suggestion
func (w *Writer) Write(key zed.Value, count uint64, offset int64) error {
```